nodejs/base/Dockerfile.tmpl

```diff
@@ -0,0 +1,108 @@
+FROM unocha/alpine-base-s6:%%UPSTREAM%%
+
+# Parse arguments for the build command.
+ARG NODE_VERSION=6.12.0
+ARG YARN_VERSION=1.3.2
+ARG VERSION
+ARG VCS_URL
+ARG VCS_REF
+ARG BUILD_DATE
+
+# A little bit of metadata management.
+# See http://label-schema.org/
+LABEL org.label-schema.schema-version="1.0" \
+      org.label-schema.build-date=$BUILD_DATE \
+      org.label-schema.vendor="UN-OCHA" \
+      org.label-schema.version=$VERSION \
+      org.label-schema.vcs-url=$VCS_URL \
+      org.label-schema.vcs-ref=$VCS_REF \
+      org.label-schema.name="nodejs-base" \
+      org.label-schema.description="This service provides a base nodejs platform." \
+      org.label-schema.architecture="x86_64" \
+      org.label-schema.distribution="Alpine Linux" \
+      org.label-schema.distribution-version="3.6" \
+      info.humanitarianresponse.nodejs="$NODE_VERSION"
```

Comment:
Probably also stick in the yarn version.

Reply:
Good idea!